packages/universal-components/src/CarrierLogo/CarrierLogoHelpers.js

```diff
@@ -9,18 +9,23 @@ export const getCarrierImageUri = (
   pixelRatio: number,
   carrierData: CarrierData,
 ): string => {
+  const type =
+    carrierData.type === undefined
+      ? CARRIER_TYPE_OPTIONS.AIRLINE
+      : carrierData.type;
```

Comment:
any reason for not doing `const type = carrierData.type ?? CARRIER_TYPE_OPTIONS.AIRLINE`

Is it because you want to set type = null if carrierData.type === null?

Reply:
Thanks, updated. I just blindly extracted this part and moved it few lines up from url constructor without thinking about it. It should fall to default value also for `null` case, so this version is definitely better and shorter as a bonus.